cv32e40s/tests/programs/custom/test_m/test_m.c

```diff
@@ -0,0 +1,182 @@
+
```

Comment:
There should probably be a license header here.

See this file for a reference: https://github.com/openhwgroup/core-v-verif/blob/6d4d5d887b26a43df2bc8306b7ec08bef63f111c/cv32e40s/tests/programs/custom/clic/clic.c

(I realize that "b_ext_test" also lacks a license header.)